src/Momento.Sdk.Incubating/Requests/CollectionUpdateTtl.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+namespace Momento.Sdk.Incubating.Requests
+{
+    public record struct CollectionUpdateTtl(TimeSpan? ttl = null, bool refreshTtl = true)
+    {
```

Comment:
Can you add a couple statics
```
public static readonly CollectionUpdateTtl FROM_CONFIGURATION = CollectionUpdateTtl(null, true);
public static readonly CollectionUpdateTtl FROM_CONFIGURATION_WITHOUT_REFRESH = CollectionUpdateTtl(null, false);
```

Then you could make the default TTL behavior in the APIs explicit and discoverable. I.e., instead of a parameter
```
CollectionUpdateTtl? ttl = null
```
which nobody can look at and understand what it means, you could do
```
CollectionUpdateTtl ttl = FROM_CONFIGURATION
```
which is much easier to understand, and you can attach `///` comments to it to explain any subtleties in a user's IDE.